R/cal-estimate-beta.R

```diff
@@ -34,8 +34,13 @@ cal_estimate_beta.data.frame <- function(.data,
                                          location_params = 1,
                                          estimate = dplyr::starts_with(".pred_"),
                                          parameters = NULL,
-                                         ...) {
+                                         ...,
+                                         group = NULL) {
   stop_null_parameters(parameters)
+
+  check_group_argument({{ group }}, .data)
+  .data <- dplyr::group_by(.data, dplyr::across({{ group }}))
+
```

Comment:
All `*.data.frame()` functions calls `check_group_argument()` and applies `dplyr::group_by()` with the group. This does nothing if `group = NULL`